apiserver/Dockerfile.apiserver

```diff
@@ -1,10 +1,11 @@
 FROM {{ hail_base_image.image }}
 
 COPY apiserver/apiserver /apiserver
-COPY hailjwt/setup.py /hailjwt/
-COPY hailjwt/hailjwt/ /hailjwt/hailjwt/
-RUN python3 -m pip install --no-cache-dir /hailjwt \
-  && rm -rf /hailjwt
+
+COPY gear/setup.py /gear/
```

Comment:
I don't see a setup.py in /gear

Reply:
Also, if this setup.py specifies find_packages(), [empty __init__.py is needed in the gear/gear, or hailjwt's setup.py will need to change to state `packages=find_namespaced_packages()`](https://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages)